content/blog/err_cert_authority_invalid_error.md

```diff
@@ -0,0 +1,232 @@
+outputs:
+  - HTML
+  - AMP
+---
+
+The **NET::ERR_CERT_AUTHORITY_INVALID** error can be a frustrating roadblock for both website visitors and owners. This common SSL/TLS certificate issue disrupts trust and hampers website accessibility. In this blogs, we’ll explain what this error means, its implications for users and website owners, and the steps to resolve it effectively.
+
+{{% tldr %}}
+1. **What is `NET::ERR_CERT_AUTHORITY_INVALID`?**  
+   This error indicates the browser cannot verify the authenticity of a website's SSL/TLS certificate.  
+   - **For Website Visitors**: It can result from an expired certificate, an untrusted Certificate Authority (CA), or outdated browser settings.  
+   - **For Website Owners**: It signals issues such as self-signed certificates, expired certificates, domain mismatches, or missing intermediate certificates.  
+
+2. **How to Fix It?**  
+   - **For Website Visitors**:  
+     - Check the URL for typos or phishing attempts.  
+     - Clear SSL state:  
```

Comment:
Add the point as 2nd point: "Check system date / time settings"
Bring the point as 3rd point: "Update your browser and try a different device or network."